app-conf/GeneralConf.xml

```diff
@@ -32,4 +32,9 @@
     <value>60000</value>
     <description>Interval between retries in milliseconds</description>
   </property>
+  <property>
+    <name>drelephant.application.search_matches_partial</name>
```

Comment:
to keep it consistent with other property names, I would suggest drelephant.application.search.match.partial as name.